.travis.yml

```diff
@@ -8,10 +8,22 @@ install:
   - go get -u github.com/golang/lint/golint
   - go get -u github.com/Masterminds/glide
   - go get -u golang.org/x/net/context
+  - go get -u github.com/HewlettPackard/gas
   - export GO15VENDOREXPERIMENT=1
   - glide install
 
 script:
+  # GAS commands for SDL scanning
+  - (cd arm && gas -fmt=json -out=results.json ./... > /dev/null)
```

Comment:
How about just  just the following (assuming `gas` is capable of printing stuff to stdout just fine):

    gas -fmt=json ./arm | tee results.json
    test $(grep -c "\"severity\":\s*\"HIGH\"" results.json) -lt 1

you can keep overwriting the results.json, it is fine because if it fails once, it will not proceed. and if it succeeds you don't need the prior files.

But I'm generally -1 for listing each pkg separately and saying `-lt 1` , `-lt 2`. I don't currently know why they are 1 or 2. Why don't we just fix them and make it zero?